### DISCUSSION TITLE
Hide Tiddler Body Programmtically

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6089] [Ideas]
Refereeing to:

1. https://talk.tiddlywiki.org/t/hide-tiddler-body/1049
2. https://talk.tiddlywiki.org/t/section-editor-plugin-first-stable-release/1060

and the new features in TW 5.1.23+ (specially 5.2.0), it is easy to write parser in wikitext and create powerful tools!

However, to use a custom parser like the simple [Section Editor](https://talk.tiddlywiki.org/t/section-editor-plugin-first-stable-release/1060), one needs to hide the tiddler body and display what plugin produces for tiddler.

There are two ways to hide tiddler body right now:
1. add hide-body field manually to any tiddler
2. use a dynamic stylesheet (https://talk.tiddlywiki.org/t/hide-tiddler-body/1049/8?u=mohammad), this is dynamic but creates huge stylesheet over the time

I want to know if we can have a simple and efficient dynamic solution to hide the tiddler body!

**Comment by kookma** [comment]
I think Streams by @saqimtiaz also hide the tiddler body!

**Comment by saqimtiaz** [comment]
@kookma I recommend reading #5136 which discusses related issues and possible solutions.

**Reply by kookma** [reply]
@saqimtiaz I read the https://github.com/Jermolene/TiddlyWiki5/issues/5136 thoroughly, still I have not found how to hide tiddlers body programmatically! I do not want to touch core tiddlers! Now I use manual adding field hide-body  = yes / conditional css to hide tiddler body!

**Reply by saqimtiaz** [reply]
@kookma the issue discusses possible solutions to implement for the core but none of those are currently possible. However reading the issue should have highlighted that this is a larger and deeper change that is needed and will need proper consideration.
